### DISCUSSION TITLE
Move website to a CDN?

### DISCUSSION THREAD

**Discussion by itrich** [SovereignCloudStack/website#300] [Ideas]
In order to optimize the speed of our website we should move the content to a CDN. There are some OSS-based alternatives like @keyCDN. What's your opinion on this?

**Comment by berendt** [comment]
Would suggest to do a time measurement to the current website first. Optimizing something without details makes no sense IMO.

**Reply by itrich** [reply]
Running `Lighthouse` tests on <https://web.dev> gives us a performance index of 69/100 due to ~5s time to interactive. I'm not absolutely sure whether a CDN would help here (or if we can optimize further on our own), but it would definitely increase the performance of oversea connections.

**Reply by berendt** [reply]
We should first find out what causes the delay. Usually a CDN only fixes the symptoms a bit but not the actual problem.

**Reply by itrich** [reply]
I created an issue adressing the performance problems. See #306. We should work on the four items in the given order.
